jormungandr/src/network/mod.rs

```diff
@@ -77,7 +77,10 @@ use std::sync::Arc;
 use std::time::Duration;
 
 pub use self::bootstrap::Error as BootstrapError;
+use crate::network::convert::TryFromNetwork;
 use crate::stats_counter::StatsCounter;
+use chain_network::data::block::try_ids_from_iter;
```

Comment:
There will be also `fragment::try_ids_from_iter`, so it's better to import the `block` module and disambiguate with the module name.